apps/workshop/test.sh

```
@@ -59,6 +62,24 @@ for client_id in workshop-test-client workshop-rotated-client; do
     node apps/workshop/tests/oauth-sync.integration.mjs
 done
 
+OPENROUTER_API_KEY= PB_EXPOSE_PORT="$WORKSHOP_TEST_PORT" docker compose -p "$WORKSHOP_TEST_PROJECT" \
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)